app/views/admin/configs/_apple_auth_provider_settings.html.erb

```diff
@@ -5,7 +5,7 @@ requires a custom partial
 -->
 <fieldset class="config-authentication-form">
   <div
-    class="crayons-notice crayons-notice--warning auth-warning <%= provider_keys_configured?(provider.provider_name) ? "hidden" : "" %>">
+    class="crayons-notice crayons-notice--warning auth-warning <%= authentication_enabled_providers.include?(provider.provider_name) ? "hidden" : "" %>">
```

Comment:
Since we already have a nice `AuthenticationHelper` module, do we maybe want to add a method like `authentication_provider_enabled?(provider_name)` there instead of leaking an implementation detail into the view?

We could change related code in a follow-up PR.

Reply:
I would be down for sending in a follow up PR with this refactor. `provider_keys_configured?` used to have that role if I'm not mistaken, I can see a few places where `authentication_enabled_providers.include?(provider_name)` is used.

Just to confirm, the implementation of this suggested `authentication_provider_enabled?(provider_name)` would possibly be as simple as wrapping this same statement right? At least this way the views would call a helper method (with test coverage on it) instead of placing these statements in the views, does this make sense or sounds useful?

```ruby
def authentication_provider_enabled?(provider_name)
   authentication_enabled_providers.include?(provider_name)
end
```

Reply:
Yes, that's what I meant. 

I think views should "know" as little as possible, which includes knowing if something will respond to a certain message like `include?` here. As unlikely as it is if we ever change the underlying implementation (e.g. `Authentication::Providers.const_get(klass).enabled?`), we shouldn't have to update views.